### PR TITLE
feat: hide compliance events (FPLA-2116)

### PR DESCRIPTION
**Pull request by joez-f**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/FPLA-2116

### Change description ###

Hide compliance events from users and remove e2e tests

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: ccd-definition/CaseEvent/CareSupervision/DEPRECATED.json [hunk 1]
```diff
@@ -472,5 +472,75 @@
...
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "ID": "COMPLY_LOCAL_AUTHORITY",
+    "Name": "Comply with directions",
+    "Description": "Allows Local Authority user access to comply with their directions as well as ones for all parties",
+    "DisplayOrder": 11,
+    "PreConditionState(s)": "PREPARE_FOR_HEARING;ISSUE_RESOLUTION;FINAL_HEARING",
+    "PostConditionState": "*",
+    "SecurityClassification": "Public",
+    "CallBackURLAboutToStartEvent": "${CCD_DEF_CASE_SERVICE_BASE_URL}/callback/comply-with-directions/about-to-start",
```
Comment (lewisbirks) [inline ccd-definition/CaseEvent/CareSupervision/DEPRECATED.json:486]: I would remove the end points as they are no longer required, maybe the endbutton label as well. Ideally all that needs to be left here is the bare minimum definition of the event

**Review by lewisbirks — COMMENTED** [review]
Are you going to remove the case event to fields/complex type definitions. What about the java code as well?

**Comment by joez-f** [comment]
> Are you going to remove the case event to fields/complex type definitions. What about the java code as well?

planning on removing definitions and code in a following cleanup PR, wanted to keep this one small

File: ccd-definition/CaseEvent/CareSupervision/DEPRECATED.json (hunk 1 above)
Comment (joez-f) [reply ccd-definition/CaseEvent/CareSupervision/DEPRECATED.json:486]: Sure

**Comment by lewisbirks** [comment]
> > Are you going to remove the case event to fields/complex type definitions. What about the java code as well?
> 
> planning on removing definitions and code in a following cleanup PR, wanted to keep this one small

If you wanted to, once this is merged you could ask for this to be run
```sql
select event_id, count(event_id)
from case_event
where event_id in ('COMPLY_LOCAL_AUTHORITY', 'COMPLY_CAFCASS', 'COMPLY_COURT', 'COMPLY_OTHERS', 'COMPLY_ON_BEHALF_COURT') and case_type_id='CARE_SUPERVISION_EPO'
group by event_id;
```
you could see if they have actually been used at all and then remove them fully if not.

**Review by lewisbirks — APPROVED** [review]

**Pull request merged**